Gems/Atom/RHI/Code/Source/RHI/RHISystem.cpp

```diff
@@ -270,19 +288,14 @@ namespace AZ
 
         ConstPtr<PlatformLimitsDescriptor> RHISystem::GetPlatformLimitsDescriptor() const
```

Comment:
lets add a default device index (defaulted to 0) for this method as well.

Reply:
I wonder how device specific `PlatformLimitsDescriptor` is/should be? Maybe it would make sense to split it up? E.g., separate out the `PlatformDefaultValues`. Some parts appear to be generic/global, while other parts might depend on the backend or on the device?